Data/3_0/Skills/act_str.lua

```diff
@@ -3547,6 +3557,13 @@ skills["MoltenStrike"] = {
 		area = true,
 	},
 	baseMods = {
+		skill("projectileSpeedAppliesToMSAreaOfEffect", true),
+		skill("radius", 9, { type = "SkillPart", skillPart = 2 }),
+		skill("radiusLabel", "Ball area:", { type = "SkillPart", skillPart = 2 }),
+		skill("radiusSecondary", 2, { type = "SkillPart", skillPart = 2 }),
```

Comment:
The minimum distance for Molten Strike's projectiles isn't affected by AoE modifiers, or at least it wasn't in the past(and it would be a relatively big change if it did so I'm guessing it would've been in patch notes at some point): http://www.pathofexile.com/forum/view-thread/916639/page/2#p7790589